schema/codelists/interestType.csv

```diff
@@ -1,13 +1,14 @@
-code,title,description,technical note
-shareholding,Shareholding
-voting-rights,Voting rights
-appointment-of-board,Appointment of board
-influence-or-control,Influence or control
-senior-managing-official,Senior managing official
-settlor-of-trust,Settlor of trust
-trustee-of-trust,Trustee of a trust.
-protector-of-trust,Protector of a trust
-beneficiary-of-trust,Beneficiary of a trust
-other-influence-or-control-of-trust,Other influence or control of a trust
-rights-to-surplus-assets,Rights to surplus assets
-rights-to-profit-or-income,Rights to receive profits or income
\ No newline at end of file
+code,title,description,technical note,
+shareholding,Shareholding,"An economic interest in an entity gained by holding shares.",,
+voting-rights,Voting rights,"A controlling interest in an entity gained by holding shares. Defined as the right of shareholders to vote on matters of corporate policy, including decisions on the makeup of the board of directors, issuing securities, initiating corporate actions and making substantial changes in the corporation's operations.",,
+appointment-of-board,Appointment of board,"A controlling interest in an entity. Defined as the absolute right to appoint members of the board of directors.",,
+other-influence-or-control,Other influence or control, "Any influence or control in an entity that is distinct from being a shareholder, having voting rights or having the absolute right to appoint to the board.",,
+senior-managing-official,Senior managing official,"A controlling interest in an entity gained by employment. Defined as the person who exercises control over the management of the entity. In the absence of a natural person(s) that exercises control of the Entity through ownership interests, the Senior Managing Official can be identified as the Controlling Person(s) of the Entity.",,
```

Comment:
"A controlling interest in an entity gained by employment. Defined as the person who exercises control over the management of the entity. <s>In the absence of a natural person(s) that exercises control of the Entity through ownership interests, the Senior Managing Official can be identified as the Controlling Person(s) of the Entity.</s>"

This last bit is specific to a disclosure regime.

Reply:
Edited as per suggestion